runbot_traefik/controllers/main.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from openerp.addons.runbot.runbot import RunbotController
+except ImportError:
+    _logger.info('Cannot find the `runbot` module in addons path.')
+
+
+class RunbotController(RunbotController):
+
+    def build_info(self, build):
+        build_info = super(RunbotController, self).build_info(build)
+        if build.repo_id.is_traefik:
+            build_info.update({
+                'host': build.repo_id._domain(),
```

Comment:
Note to self: this is used in the template to show the worker, so that will need to be updated in the template
![image](https://user-images.githubusercontent.com/7397578/28651614-bb3a42bc-7237-11e7-99c7-12c6c10e979e.png)